about.html

```diff
@@ -0,0 +1,150 @@
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About us</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header id="header">
+        <div class="header-top">
+            <div class="wrapper">
+                <nav>
+                    <ul class="top-nav">
+                        <li><img src="images/icons/fb-icon.svg"></li>
+                        <li><img src="images/icons/twitter-icon.svg"></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+
+        <div class="header-main">
+            <div class="wrapper">
+                <nav class="main-nav">
+                    <div class="icon-nav">
+                        <a href="index.html"><img src="images/logo.jpg" alt="logo image"></a>
+                    </div>
+                    <div class="icon-menu-mobile">
+                        <img src="images/icons/hamburger-icon.svg" alt="Hamburger icon">
+                    </div>
+                    <ul class="bottom-nav">
+                        <li><a  class="active" href="about.html">About</a>
+                        <li><a href="#logo">Mission</a></li>
+                        <li><a href="#past-events">Events</a></li>
+                        <li><a href="#partners">Partners</a></li>
+                        <li><a href="#">Blog</a></li>
+                        <li><button type="button">Follow US</button></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+
+        <div class="nav-mobile">
+            <div class="cross">
+                <img src="images/icons/cross.svg" alt="cross">
+            </div>
+            <ul class="container">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#logo">Mission</a></li>
+                <li><a href="#past-events">Events</a></li>
+                <li><a href="#partners">Partners</a></li>
+                <li><a href="#">Blog</a></li>
+            </ul>    
+        </div>
+    </header>
+
+    <main>
+        <section id="headline">
+            <div class="wrapper headline-content t-center">
+                <h1>A mission to share a common Goal</h1>
+                <div class="headline-about-description">
+                    <p>Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.
+
+                        Donations to freeCodeCamp go toward our education initiatives, and help pay for servers, services, and staff.</p>
```

Comment:
- [ ] In my opinion, this section must stick to the design too, by reducing the opacity of the background and using a dark border for the content text as shown below:

your design:
![Yabout](https://user-images.githubusercontent.com/97834160/211112477-6e3bd2eb-9e32-4457-b86a-3f4974ca6d7d.png)
Behance design:
![Babout](https://user-images.githubusercontent.com/97834160/211112497-2a813fb8-1b68-4dd8-aaef-0ce934970945.png)

Reply:
✅ Configured According to the design.